R/msaGaugeRR.R

```diff
@@ -167,7 +167,7 @@ msaGaugeRR <- function(jaspResults, dataset, options, ...) {
     TrafficContainer <- jaspResults[["trafficPlot"]]
 
     valuesVec <- .gaugeANOVA(dataset = dataset, measurements = measurements, parts = parts, operators = operators, options =  options, ready = ready, returnTrafficValues = TRUE)
-    TrafficContainer[["plot"]] <- .trafficplot(StudyVar = valuesVec$study, ToleranceVar = valuesVec$tol, options = options, ready = ready)
+    TrafficContainer[["plot"]] <- .trafficplot(StudyVar = valuesVec$study, ToleranceUsed = options$gaugeToleranceEnabled,ToleranceVar = valuesVec$tol, options = options, ready = ready)
```

Comment:
you're using PascalCase here, but in jasp, we use camelCase (see e.g., https://techterms.com/definition/pascalcase). Did you do this intentionally?